test/e2e/deployment.go

```diff
@@ -149,6 +152,23 @@ func checkDeploymentRevision(c *clientset.Clientset, ns, deploymentName, revisio
 	return deployment, newRC
 }
 
+func stopDeployment(c *clientset.Clientset, oldC client.Interface, ns, deploymentName string) {
+	deployment, err := c.Extensions().Deployments(ns).Get(deploymentName)
+	Expect(err).NotTo(HaveOccurred())
+
+	Logf("stopping deployment %s", deploymentName)
```

Comment:
Suggest that we use the term "delete" instead of "stop" since `kubectl stop` is deprecated and superseded by `delete`.